nav2_smac_planner/lattice_primitives/README.md

```diff
@@ -0,0 +1,168 @@
+# Lattice Primitive Generator
+## Contents
+
+- **[About](#about)**
+- **[Setup](#setup)**
+- **[Usage](#usage)**
+- **[Parameters](#parameters)**
+- **[Output file structure](#output-file-structure)**
+- **[How it Works](#how-it-works)**
+</br>
+
+## About
+The scripts in this folder are used to generate the motion primitives for the state lattice planner. An example of the trajectories for a grid resolution of 5cm and turning radius of 0.5m is shown below.
```

Comment:
Add the phrase "minimum control set" since this isn't just a set of arbitrary motion primitives, but a principled approach to find them to fully describe the capabilities of the motion models.

Reply:
I'd move the citation about the paper this implements up to here as well so its clear from the onset